### DISCUSSION TITLE
Pricing

### DISCUSSION THREAD

**Discussion by rcoops** [ReviewNB/support#104] [Q&A] [answered]
Hi,
Our organisation is evaluating this app (considering a business account) and we have a question on pricing:

From first glance, it looks like pricing is based on repos/users but you add the app to the org. We have a number of repos and users but only a subset of each that use notebooks - we couldn't justify going Enterprise. Can you be selective about which users and/or repos this app is added to or is it an all or nothing on the org?

**Comment by amit1rrr** [comment, marked as answer]
> Can you be selective about which users and/or repos this app is added to or is it an all or nothing on the org?

Yes, you can be selective. After installation, you get to select the specific Repositories on which you want to enable ReviewNB. See screenshot below. ReviewNB GitHub application is restricted to the repositories you select.

For User selection - we don't have an admin UI for User management as yet. Only active users are counted towards the plan limit. In practice, you will not face any issues unless the number of Users actively using the service consistently cross the plan limit.

Hope this helps. Let me know if you've any questions.


![5f3795aa598ce65f47077816_repo_selection_compressed-p-800 (1)](https://user-images.githubusercontent.com/1350490/150974661-32fbf2bc-36cd-4163-aa6c-13bc24672cc3.png)

**Reply by rcoops** [reply]
Excellent - and just to confirm only _selected_ repositories are counted toward the billing model?

**Reply by amit1rrr** [reply]
We actually do better than that. Intersection of (Selected + Actively Used) repositories are counted towards the plan limit.

E.g. On the Business plan you can select say 20+ private repositories but you're well within the plan limits as long as 10 or less repositories are being used in a given calendar month.

This model gives you freedom to connect as many repositories as you like & only pay for the ones you actually use. Hope this helps.

**Reply by rcoops** [reply]
Two more questions:
Does 'active users' include both PR submitters and reviewers?
Is it the same measure for active users as with repos (e.g. business plan => 30 active users within a calendar month)

**Reply by amit1rrr** [reply]
> Does 'active users' include both PR submitters and reviewers?

Active user is anyone who uses ReviewNB to view their PR / Commit / Notebooks. They could be PR submitter / reviewer / both / neither.  If they use the service in a given month, they're an active user.

> Is it the same measure for active users as with repos (e.g. business plan => 30 active users within a calendar month)

Yes.

**Reply by evelyn-mark** [reply]
Hello all, 
thank you for the informative thread. 

Since my team's code review process already relies on this tool I'm trying to understand more about the **active user** counting mechanism & followingly the pricing policy schemas.  So I'm not 100% clear on the following:

> Active user is anyone who uses ReviewNB to view their PR / Commit / Notebooks. They could be PR submitter / reviewer / both / neither. If they use the service in a given month, they're an active user.

For example, do the watchers also count, since they may potentially also view a PR? Could you please provide some more details?
Is there also a way to identify the current users that are considered active?

**Reply by amit1rrr** [reply]
> do the watchers also count, since they may potentially also view a PR? Could you please provide some more details?

If a user opens a PR (or any other page) in the ReviewNB UI, they're counted as an active user for that org.

> Is there also a way to identify the current users that are considered active?

You can write to support@reviewnb.com from the official email address & we can provide you with the current list of active users on the account.
